rest/admin_api.go

```diff
@@ -153,6 +153,39 @@ func (h *handler) handleReplicate() error {
 		return err
 	}
 
+	client := &http.Client{}
+	targetRequest, err := http.NewRequest("GET", params.GetTargetDbUrl(), nil)
```

Comment:
- Only generally need to create a new request when using custom headers. Let's switch to use the `.Get()` method on http.Client.
- Will need to close the resp.Body on non-error.
- Check for `resp.StatusCode >= 400` instead - otherwise we'll error for redirects, etc.

```suggestion
	resp, err := h.server.HTTPClient.Get(params.GetTargetDbUrl())
	if err != nil {
		// TODO: handle
	}

	defer resp.Body.Close()
	if resp.StatusCode >= 400 {
		// TODO: handle
	}
```